pkg/sdk/testint/row_access_policies_gen_integration_test.go

```diff
@@ -350,3 +350,60 @@ func TestInt_RowAccessPoliciesShowByID(t *testing.T) {
 		require.Equal(t, id2, e2.ID())
 	})
 }
+
+func TestInt_RowAccessPoliciesDescribe(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	cleanupRowAccessPolicyHandle := func(id sdk.SchemaObjectIdentifier) func() {
+		return func() {
+			err := client.RowAccessPolicies.Drop(ctx, sdk.NewDropRowAccessPolicyRequest(id))
+			if errors.Is(err, sdk.ErrObjectNotExistOrAuthorized) {
+				return
+			}
+			require.NoError(t, err)
+		}
+	}
+
+	createRowAccessPolicyHandle := func(t *testing.T, id sdk.SchemaObjectIdentifier, args []sdk.CreateRowAccessPolicyArgsRequest) {
```

Comment:
we can use the creation from the test client heper here

Reply:
Done.